design/noobaa/system-read.md

```diff
@@ -0,0 +1,67 @@
+♜ [Rook NooBaa Design](README.md) /
+# System Read
+
+The operator will set the status of the NooBaa system to represent the current state of reconciling to the desired state.\
+Here is the example status structure as would be returned by a `kubectl get noobaa <system-name> -o yaml`:
+
+```yaml
+apiVersion: noobaa.rook.io/v1alpha1
+kind: System
+metadata:
+  name: noobaa-1
+  namespace: rook-noobaa
+spec:
+    # ...
+status:
+
+  health:
+    accounts: OK
+    backingStores: OK
+    bucketPolicies: OK
+    bucketProvisioner: OK
+    issues: []
+
+  counters:
+    accounts: 3
+    backingStores: 2
+    bucketPolicies:
+      templates: 3
+    bucketProvisioner:
+      exportedStorageClasses: 3
+    buckets: 33
+  
+  readme: |
+
+Welcome to NooBaa
+
+Management UI
+-------------
+- Username/password     : admin@noobaa.io / TgerOAZ9MOd/Xg== 
```

Comment:
Will these secrets be exposed in plain text here in the CR status section? Shouldn't they remain in K8s secrets or handled differently?

Reply:
Changed so that the status will only show the command to extract the secret to make it easy for the user to get it.
Resolving.